README.md

```diff
@@ -1,6 +1,30 @@
-[![Build Status](https://travis-ci.org/njoy/NJOY21.svg?branch=master)](https://travis-ci.org/njoy/NJOY21)
-
 # NJOY21 --- NJOY for the 21st Century
-NJOY21 is the future version of NJOY. For more information please visit [https://njoy.github.io/NJOY21](gttps://njoy.github.io/NJOY21).
+NJOY21 is the modernized version of NJOY---suitable for computing in the 21st Century. It is based on [NJOY2016](https://njoy.github.io/NJOY21).
+
+## Differences between NJOY21 and NJOY2016
+NJOY21 is the modern successor to NJOY2016. It is backwards compatible with NJOY2016---except for a few small changes. 
+
```

Comment:
Maybe mention the -h option for getting help on the command line (if it exists)?